src/main/kotlin/org/jqassistant/tooling/intellij/plugin/editor/effectiveConfig/LoadingPanel.kt

```diff
@@ -0,0 +1,15 @@
+package org.jqassistant.tooling.intellij.plugin.editor.effectiveConfig
+
+import java.awt.BorderLayout
+import javax.swing.ImageIcon
+import javax.swing.JLabel
+import javax.swing.JPanel
+
+class LoadingPanel : JPanel() {
+    init {
+        layout = BorderLayout()
+        val loadingIcon = ImageIcon(javaClass.getResource("/icons/loading.gif"))
```

Comment:
```suggestion
        val loadingIcon = AnimatedIcon.Default()
```
Using the IntelliJ provided loading Icon works better with high DPI screens and is also used in the IntelliJ codebase